src/line-graph.js

```diff
@@ -34,7 +38,44 @@ export class LineGraph extends HTMLElement {
     this.renderGraph(this.parseData(data), svg);
   }
 
- 
+
+  /**
+   * Create set of distinct values
+   * @param {data} data 
+   */
+  maxValue(data) {
```

Comment:
Can we move this function to a helper class? It looks like you may have to create one, I feel our `LineGraph` class may be getting a little too large.